lib/pipe/html.js

```diff
@@ -135,18 +135,29 @@ class HtmlPipe {
         test.title = 'Unknown test title';
       }
 
-      if (!test.files || test.files.length === 0) {
-        test.files = 'This test has no files';
+      if ('files' in test) {
```

Comment:
why not just `if (!test.files?.length)` ?

Reply:
fixed